Cargo.toml

```diff
@@ -49,7 +49,7 @@ rustc-hash = "2.0.0"
 rrs-lib = "0.1.0"
 tracing = "0.1.40"
 
-womir = { git = "https://github.com/powdr-labs/womir.git", rev = "f12ea50e74a872" }
+womir = { git = "https://github.com/powdr-labs/womir.git", rev = "b4ba2b1" }
```

Comment:
can be updated now that the womir PR was merged

Reply:
done